common/src/main/java/net/william278/husktowns/hook/EconomyHook.java

```diff
@@ -14,9 +15,16 @@ protected EconomyHook(@NotNull HuskTowns plugin, @NotNull String name) {
 
     public abstract boolean hasMoney(@NotNull OnlineUser user, @NotNull BigDecimal amount);
```

Comment:
Do we still need `hasMoney`? If not, get rid of it. I assume we probably still do if it's used elsewhere.

Reply:
nope. i also removed an unused getBalance